Dfe.Academies.External.Web/Pages/ApplicationOverview.cshtml.cs

```diff
@@ -0,0 +1,41 @@
+using Dfe.Academies.External.Web.Models;
+using Dfe.Academies.External.Web.Pages.Base;
+using Dfe.Academies.External.Web.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Dfe.Academies.External.Web.Pages;
+
+public class ApplicationOverviewModel : BasePageModel
```

Comment:
I'd prefer to have kept this out of this PR on the What Is Your Role page, just because it makes the PR bigger than it needs to be. Not worried about it now but just a thought.